package-lock.json

```diff
@@ -10,9 +10,11 @@
       "license": "ISC",
       "dependencies": {
         "body-parser": "^1.19.2",
+        "cors": "^2.8.5",
```

Comment:
Package-lock.json should be removed from repository and added to gitignore

Reply:
I added package-lock.json to gitignore